### DISCUSSION TITLE
2022-in-review/

### DISCUSSION THREAD

**Discussion by giscus** [ironicbadger/blog.ktz.me#215] [Announcements]
# 2022-in-review/

A wedding. Lots of travel. And lots of friends.

https://blog.ktz.me/2022-in-review/

**Comment by darrenwjlau** [comment]
Wow

**Comment by gardncol** [comment]
This is great. How did you collate all of these events into a list?

**Comment by sillmnvg** [comment]
Can't wait to hear about the new Megadesk project.
